index.html

```diff
@@ -0,0 +1,294 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="./css/main.css" />
+  <link rel="stylesheet" href="./css/menu.css" />
+  <link rel="icon" href="./favicon.ico" />
+  <title>Aziz Mejri - Portfolio</title>
+</head>
+
+<body>
+  <header>
+    <h1 id="logo">Aziz Mejri</h1>
+    <button type="button" class="menu-bars">
+      <i class="fas fa-bars"></i>
+    </button>
+    <nav id="desktop-nav">
+      <ul>
+        <li><a class="nav-link" href="#projects">Portfolio</a></li>
+        <li><a class="nav-link" href="#about-me">About</a></li>
+        <li>
+          <a class="nav-link" href="#contact">Contact <i class="far fa-envelope"></i></a>
+        </li>
+      </ul>
+    </nav>
```

Comment:
The mobile menu in the user interface does not follow the Figma design. kindly refer [Here](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=2-9) to use the required design